### PR TITLE
Ewl 3477 hub add cta 50 50 video component to styleguide

### PR DESCRIPTION
**Pull request by chelsiejohnston**
## JIRA Ticket(s)
- [EWL-3477: hub add cta 50 50 video component to styleguide](https://issues.ama-assn.org/browse/EWL-3477)

## Description:
Adds the following:
- Atom: "Video Youtube"
- Molecule: "Hub CTA 50/50 Video"
- Hub CTA 50/50 Video molecule as a newly included pattern in the "Hub" Template
- A JS file, hub-cta-half-video.js, where `.fitvids()` is initialized

The most important thing to do in this ticket was prevent the video and thumbnail from being cropped at any viewpoint. @froboy came up with the solution in the attached screenshot, which uses fitvids() to scale the video responsively. Fitvids.js was included in the codebase already, but was not used in the style guide up to this point.

## To Test:
- Navigate to Atoms > Media > Video - Youtube
- Observe that the Media - Youtube atom exists
- Navigate to Molecules > Hub Components > Hub CTA 50/50 Video
- Observe that the Hub CTA 50/50 Video molecule exists and scales responsively as shown in the screenshot 
- Navigate to Templates > Hub > Hub and scroll until you see the Hub CTA 50/50 Video molecule included; verify that it scales responsively as shown in the screenshot 

## Automated Test:
N/A

## Remaining Tasks:
N/A

## Additional Notes:
N/A

---

[Pull Request Guidelines](https://github.com/palantirnet/development_documentation/blob/master/guidelines/pr_review.md)

### CODE REVIEW

File: styleguide/source/_patterns/00-atoms/08-media/01-video-youtube/01-video-youtube.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+el: ".test"
+title: "Video - Youtube"
+---
+Youtube video embedded player PLACEHOLDER DESC
```
Comment (froboy) [inline styleguide/source/_patterns/00-atoms/08-media/01-video-youtube/01-video-youtube.md]: Please put desc in here.

File: styleguide/source/_patterns/01-molecules/09-hub-components/01-hub-cta/hub-cta-half-video.twig [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<br>
```
Comment (froboy) [inline styleguide/source/_patterns/01-molecules/09-hub-components/01-hub-cta/hub-cta-half-video.twig]: Please remove this br

**Comment by chelsiejohnston** [comment]
This is ready to be reviewed.

**Comment by froboy** [comment]
LGTM. Thanks for the updates.

**Pull request merged**